BRAD/endpoints.py

```
@@ -83,6 +83,7 @@
 import shutil
 import logging
 import time
+from itertools import filterfalse
```

Ruff

`itertools.filterfalse` imported but unused

Remove unused import: `itertools.filterfalse`

(F401)